packages/larva-patterns/components/c-icon/c-icon.twig

```diff
@@ -3,7 +3,11 @@
 	<span class="lrv-a-screen-reader-only">{{ c_icon_link_screen_reader_text }}</span>
 {% endif %}
 
-<svg class="c-icon {{ modifier_class }} {{ c_icon_classes }}">
+{% if c_icon_screen_reader_text %}
+	<abbr class="lrv-a-screen-reader-only" title="{{ c_icon_screen_reader_title_attr }}">{{ c_icon_screen_reader_text }}</abbr>
```

Comment:
In other patterns, screen reader text is added in a `span` element. Is there a specific reason to use `abbr` here instead of span? As far as I know, the text will not necessarily be an abbreviation, so `span` usually makes the most sense semantically.

Reply:
That was requirement in the PMCP-2505 Jira ticket. So the same changes I have made here.

Let me know if that still need to change that.

Reply:
Ah I understand - we do not want `abbr` for every icon, only for the PMC icon specifically. In this case, you can do the same approach as with `c-label` where you use a new value for the `c_icon_screen_reader_tag_text` - the else condition can default to `span`and the value can be updated to `abbr` when it is added in a specific context.